azureadps-1.0/MSOnline/Get-MsolDevice.md

```diff
@@ -88,6 +97,20 @@ PS C:\>Get-MsolDevice -RegisteredOwnerUpn "pattifuller@contoso.com"
 
 This command gets all the devices registered by the user with the UPN named pattifuller@contoso.com.
 
+### Example 7: Get device by activity (logon) timestamp
+```
+PS C:\>Get-MsolDevice -LogonTimeBefore 'January 1, 2017 12:00:00 AM'
+```
+
+Ths command gets all the devices with the ApproximateLastLogonTimestamp before January 1, 2017
+
+### Example 8: Get devices and include system managed devices
+```
+PS C:\>Get-MsolDevice -all -IncludeSystemManagedDevives
```

Comment:
Typo. -IncludeSystemManagedDevives should read -IncludeSystemManagedDevices (with a 'c' in last word)

Reply:
Also, -All should be capitalized.